packages/runtime-core/src/scheduler.ts

```diff
@@ -92,11 +92,8 @@ export function queueJob(job: SchedulerJob) {
     job !== currentPreFlushParentJob
   ) {
     const pos = findInsertionIndex(job)
-    if (pos > -1) {
-      queue.splice(pos, 0, job)
-    } else {
-      queue.push(job)
-    }
+    queue.splice(pos, 0, job)
```

Comment:
It's worth adding a short comment saying pos is always > -1

Reply:
I think the `findInsertionIndex` function name can explain this behavior well 😄

Reply:
Not at all as you explained in the issue that findInsertinIndex could return `-1` and the name doesn't indicate anything about it being valid. Same with vanilla functions like `findIndex` which do return -1

The comment should be added for clarity

Reply:
ok😁